### PR TITLE
[FIX] account: Change journal entry date when current date not in period

### PR DESCRIPTION
**Pull request by pedrobaeza**
As journal entries in v8 takes the invoice date, if you forced another period for that entry, when moving to v9 you will have an incorrect date on the journal entry, and that is now the only reference, so we need to change that date to for example the start date of the forced period.

cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Ready to be reviewed.

**Review by sergio-teruel — APPROVED** [review]

File: addons/account/migrations/9.0.1.1/post-migration.py [hunk 1]
```diff
@@ -717,6 +742,7 @@ def migrate(env, version):
     fill_move_line_invoice(cr)
     merge_invoice_journals(env)
     update_account_invoice_date(cr)
+    update_move_date(cr)
```
Comment (MiquelRForgeFlow) [inline addons/account/migrations/9.0.1.1/post-migration.py:745]: Shouldn't be this line before than the previous one? :thinking:

Comment (pedrobaeza) [reply addons/account/migrations/9.0.1.1/post-migration.py:745]: It doesn't matter

Comment (MiquelRForgeFlow) [reply addons/account/migrations/9.0.1.1/post-migration.py:745]: Ok

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**